src/app/pages/storage/modules/pool-manager/components/pool-manager-wizard/components/layout-step/automated-disk-selection/automated-disk-selection.component.ts

```diff
@@ -50,9 +48,7 @@ export class AutomatedDiskSelectionComponent implements OnInit, OnChanges {
 
   protected compareSizeAndTypeWith = _.isEqual;
 
-  protected vdevLayoutOptions$: Observable<Option[]> = of([
-    { label: 'Stripe', value: CreateVdevLayout.Stripe },
-  ]);
+  protected vdevLayoutOptions$ = of<Option[]>([]);
```

Comment:
Minor thing: for better type safety and autocomplete, it could be a type of: `SelectOption<CreateVdevLayout>[]`